pages/_app.js

```diff
@@ -1,12 +1,87 @@
 import GlobalStyle from "../styles";
-import { bags } from "../components/List/data";
-import List from "./List";
+import { airlines } from "../lib/data";
```

Comment:
Naming suggestion:
```suggestion
import { airlines as initialAirlines } from "../lib/data";
```

Reply:
Uhm, this is weird. The imports listed are quite outdated, I currently have these:
`import GlobalStyle from "../styles";
import { airlines } from "../lib/data";
import { useState } from "react";
import { calculateVolume } from "@/utils";`

But I will apply the suggestion alongside the follow-up since it makes perfectly sense.